eng/CredScanSuppression.json

```diff
@@ -4,6 +4,7 @@
         {
             "placeholder": [
                 "Sanitized",
+                "%2BLsuqDlN8Us5lp%2FGdyEUMnU1XA4HdXx%2BJUdtkRNr7qI%3D",
```

Comment:
Is there any way we can change this to use one of the existing more generic placeholders?